README.md

```diff
@@ -123,6 +123,7 @@ GOOGLE_SHEETS_TEMPLATE_FILE_ID=...
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=...
 
 # for Slack integration
+SLACK_BASE_URL=https://my-workspace.slack.com
```

Comment:
Should we default to the cardinality-puzzles workspace?

Reply:
Ok, that seems fine. Question: is cardinality-puzzles invite-only, or can anyone on the Internet join?

Reply:
Hmm that's a good point. Let's leave this as is, and leave a comment that it should be the same workspace that the Slack app is installed to.

Reply:
Oh ok, I can change it back to no default by default.

I was also thinking having no default makes it easier for another team to use Small Board -- they'll see a warning that no SLACK_BASE_URL is configured and then know to configure it.